### ISSUE TITLE
Combined spectral and denoising loss function

### ISSUE THREAD

**Issue by srijang97** [BoyuanJiang/FitDiT#40]
Hi,

It would be great if you could clarify the following for me as I could not find an explanation in the paper: 

In section 3.4 you mention 'Ultimately, the over all training loss function of DenoisingDiT is a combination of the image-based frequency loss and the latent-based diffusion noise prediction loss.' I understand the loss formulations however, it would be great to know the following additional details:

1) Details of the weighting scheme used to combine the spectral and denoising loss
2) How the combined loss was introduced during training? Was it trained on the combined weighted loss from the start or was there a ramp-up period to introduce the frequency loss?

Thansk for your help!

**Comment by srijang97** [comment]
@BoyuanJiang would appreciate your guidance on this  so that I can reproduce FitDit's training method for my research
